README.md

```diff
@@ -4,7 +4,7 @@ Create and handle qemu-imgs that are suitable to perform kernel testing/developm
 
 ## Requirements
 
-`apt install qemu qemu-user-static debootstrap`
+`sudo apt install qemu-system qemu-user-static debootstrap`
```

Comment:
Split the qemu install into its own line such that we install whatever is available

sudo apt install qemu || sudo apt install qemu-system

Reply:
I''m unsure if Ubuntu has a package whose name is `qemu`:
```Bash
$ sudo apt-get install qemu
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package qemu is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'qemu' has no installation candidate
```

Reply:
It does. It seems you're on the latest and greatest. My 22.04 machine still has this package.